tests/test_dummy_converter.py

```diff
@@ -62,3 +51,55 @@ def test_default_geo_config_str() -> None:
     1    2    3
     1 1000    0
 """
+
+TEST_DIR = tempfile.TemporaryDirectory()
+
+
+def test_default_beam_config_str() -> None:
+    """Test if the default BeamConfig str representation is correct."""
+    assert str(BeamConfig()) == BEAM_STR
+
+
+def test_default_mat_config_str() -> None:
+    """Test if the default MatConfig str representation is correct."""
+    assert str(MatConfig()) == MAT_STR
+
+
+def test_default_detect_config_str() -> None:
+    """Test if the default MatConfig str representation is correct."""
+    assert str(DetectConfig()) == DETECT_STR
+
+
+def test_default_geo_config_str() -> None:
+    """Test if the default MatConfig str representation is correct."""
+    assert str(GeoConfig()) == GEO_STR
+
+
+def test_parser_run() -> None:
+    """Test if dummy parser generates output files corectly"""
+    dummy_runner = Runner(DummmyParser(), {}, TEST_DIR.name)
+    dummy_runner.run_parser()
+
+
+def test_if_beam_created() -> None:
+    """Check if beam.dat file created"""
+    with open(path.join(TEST_DIR.name, 'beam.dat')) as f:
```

Comment:
why do we expect beam.dat to be created here ?
each test function should be self-containing. 
create beam.dat in this function here or use some custom fixture if you need to create it per session

Reply:
Now we use a fixture that creates the file in a temporary directory.